hypershift-operator/main.go

```diff
@@ -217,6 +221,9 @@ func run(ctx context.Context, opts *StartOptions, log logr.Logger) error {
 	if err := hostedClusterReconciler.SetupWithManager(mgr, createOrUpdate); err != nil {
 		return fmt.Errorf("unable to create controller: %w", err)
 	}
+	if err = (&hyperv1.HostedCluster{}).SetupWebhookWithManager(mgr); err != nil {
```

Comment:
If we re-use the binary I think we should increase the default replicas to two and add node-level anti-affinity, since this is now in the path of all update operations

Reply:
I agree we should think about this.  Can that be a follow up to this PR after we agree on what we should do?

Reply:
I made this change